G/GMT/bundled/patches/win_mingw.patch

```diff
@@ -0,0 +1,66 @@
+diff --git before/gmt-6.2.0/src/gmt_api.c after/gmt-6.2.0/src/gmt_api.c
```

Comment:
This patch (minus the unrelated comments changed) should be upstreamed.

Reply:
sorry, you lost me here...

Reply:
[Upstream](https://en.wikipedia.org/wiki/Upstream_(software_development)) == Submit a pull request to https://github.com/GenericMappingTools/gmt with this change.